provisioning/service/src/Contract/IContractApiHttp.cs

```diff
@@ -13,6 +13,14 @@ namespace Microsoft.Azure.Devices.Provisioning.Service
 {
     internal interface IContractApiHttp : IDisposable
     {
+        Task<ContractApiResponse> RequestAsync(
```

Comment:
What's the reason for this interface?

Reply:
This is a generic version that can be used instead of the other ones. The idea is replace all the others by this one.

Reply:
done